src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1286,21 +1286,148 @@ export default class DataTable extends Vue {
     }
   }
 
+  _printHead() {
+    return (
+      <thead>
+        <tr>
+          {Object.keys(this.data.head).map((column: string) => {
+            return <th>{this.data.head[column].label}</th>;
+          })}
+        </tr>
+      </thead>
+    );
+  }
+
+  _printFooter() {
+    const resultsCount =
+      this.mode === DataTableModes.CLIENT
+        ? this._showSelectedOnly
+          ? this.selectedRows.length
+          : this.data.body.length
+        : this.config.pagination.results;
+
+    return (
+      <tfoot>
+        <tr>
+          <td colspan={Object.keys(this.data.head).length}>{resultsCount} results</td>
+        </tr>
+      </tfoot>
+    );
+  }
+
+  _printBody() {
+    if (this.data.body.length > 0) {
+      return (
+        <tbody>
+          {this._serializedDataBody.map((bodyRow: DataTableRow) => {
+            return this.printRow(bodyRow, 'parent');
+          })}
+        </tbody>
+      );
+    }
+    return (
+      <tbody>
+        <tr>
+          <td colspan={Object.keys(this.data.head).length} class={DATA_TABLE_CLASSES.EMPTY}>
+            {this.config.noResultsMessage ? this.config.noResultsMessage : 'No results found'}
+          </td>
+        </tr>
+      </tbody>
+    );
+  }
+
+  printRow(bodyRow: DataTableRow, rowLevel: DataTableRowLevels = 'parent') {
+    const row: any[] = [];
+    const rowCells: JSX.Element[] = [];
+    const rowAccordionContent = [];
```

Comment:
In print mode, why do we still call it accordion? Can we call it `subLevelContent` or something similar?